openmdao/core/tests/test_parallel_groups.py

```diff
@@ -308,13 +308,11 @@ def test_setup_messages_bad_vec_type(self):
 
         # check that error is thrown if not using PETScVector
         if MPI:
-            msg = ("Problem: The `distributed_vector_class` argument must be a distributed vector "
+            msg = ("Problem .*: The `distributed_vector_class` argument must be a distributed vector "
```

Comment:
I am not sure where this ".*" comes from for the problem name, but it doesn't look so great. Seems like it should be "Problem Problem1", or maybe just "Problem".  There must be a bug in the print here that wasn't uncovered until you made the change.

Reply:
My bad, that's regex.